internal/service/alertconfiguration/resource_alert_configuration_test.go

```diff
@@ -421,6 +422,32 @@ func TestAccConfigRSAlertConfiguration_withPagerDuty(t *testing.T) {
 	})
 }
 
+func TestAccConfigAlertConfiguration_PagerDutyUsingIntegrationID(t *testing.T) {
+	acc.SkipTestForCI(t) // currently skipped as third party integration id cannot be created and obtained during test
```

Comment:
Will likely be able to add this test to the CI once I can use the `third_party_integration` resource and get its id directly.

Reply:
nice!